keyboards/xd002/keymaps/rgb/keymap.c

```diff
@@ -0,0 +1,46 @@
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QMKURL = SAFE_RANGE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+    RGB_HUI,  QMKURL
+  )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  if (record->event.pressed) {
+    switch (keycode) {
+      case QMKURL:
+        // Using SEND_STRING here adds 400bytes ...
```

Comment:
:(

Reply:
There are a few other places i plan to trim space from, so one day :crossed_fingers: